### PR TITLE
Added initial priority downlink policy logic

### PR DESCRIPTION
**Pull request by bijimm**
### Description of changes:
Adding mobile sdk/demo parity for the priority downlink policy. Builders will be able to assign priority to specific remote video streams.

### Testing done:
Tested simultaneously with 2 JS SDK clients and 1 iOS client. Turned video on/off several times for the JS SDK clients to test the new iOS downlink policy.

#### Manual test cases (add more as needed):

- [x] Join meeting without CallKit
- [ ] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [x] Leave meeting
- [x] Rejoin meeting
- [ ] See metrics
- [x] See attendee presence
- [x] Send audio
- [x] Receive audio
- [x] Mute self
- [x] Unmute self
- [x] See local mute indicator when muted
- [x] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [ ] Enable and disable remote video from remote side
- [ ] Send local video
- [x] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by hensmi-amazon — APPROVED** [review]
🚢

File: AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoTileCell.swift [hunk 1]
```diff
@@ -65,6 +70,33 @@ class VideoTileCell: UICollectionViewCell {
...
+            updateVideoSubscriptionsButton.isHidden = true
+        }
+    }
+    
+    @objc func showUpdateVideoSubscriptionsMenu() {
+        let alertController = UIAlertController(title: "Set video priority", message: "Choose the display priority order for the selected video", preferredStyle: .alert)
+        let priorityList = [VideoPriority.lowest, VideoPriority.low, VideoPriority.medium, VideoPriority.high, VideoPriority.highest]
+        let titleList = ["Lowest", "Low", "Medium", "High", "Highest"]
+        
+        for i in 0...(priorityList.count-1) {
```
Comment (ziyiz-amzn) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoTileCell.swift:90]: nit: space  `count - 1`

**Review by ziyiz-amzn — APPROVED** [review]
Overall, it looks good to me.

**Review by ziyiz-amzn — APPROVED** [review]
Make sure you select `squash and merge` when merge.

**Pull request closed without merging**